typescript/infra/scripts/funding/fund-deterministic-key-from-deployer.ts

```diff
@@ -79,7 +79,8 @@ async function main() {
         logger.info(
           `Funding ${address} on chain '${chain}' with ${value} native tokens`,
         );
-        await multiProvider.sendTransaction(chain, {
+        await multiProvider.sendTransaction({
```

Comment:
I did kinda like the old interface fwiw but will defer to ppl closer to the SDK

Reply:
fair, I just felt it was cleaner to cut the arg out since it's redundant to mention `chain` twice

Reply:
I prefer the cleaner approach in my case